phoenix-core/src/main/java/org/apache/phoenix/coprocessor/GlobalIndexRegionScanner.java

```diff
@@ -883,6 +925,9 @@ public boolean verifySingleIndexRow(byte[] indexRowKey, List<Mutation> actualMut
                 logMismatch(expected, actual, expectedIndex, verificationPhaseResult, isBeforeRebuild);
             }
             else {
+                if (expected == null) {
```

Comment:
I think adding a comment here would be nice. Something like this case happens when there is extra index rows